portfolio/src/main/java/com/google/sps/servlets/LoadCommentServlet.java

```diff
@@ -77,11 +81,19 @@ public int getNumberOfComments(HttpServletRequest request) throws IOException {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    UserService userService = UserServiceFactory.getUserService();
+
+    if(!userService.isUserLoggedIn()) {
+      response.sendRedirect("/");
```

Comment:
```suggestion
      response.sendRedirect(REDIRECT_URL);
```
Nit: Can you also do this like you did in AuthServlet.java?

Reply:
Sure!